style.css

```diff
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:ital,wght@0,700;1,400&display=swap');
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  position: relative;
+  width: 100%;
+  height: 92px;
+  background: #3c3a39;
+  color: #3c3a39;
+}
+
```

Comment:
- [OPTIONAL] Kindly make sure the navigation should be aligned in the center in order to match the provided Figma design.
    _**Hint: Make sure to remove the `height` on the header, it will be aligned correctly automatically after you remove the `height`**_

     - ### YOUR DESIGN
     
    ![image](https://user-images.githubusercontent.com/87197412/235699381-42e1ec80-4fb9-493e-b9ff-78c9e5308d56.png)

     - ### AFTER THE HEIGHT REMOVAL
     
     ![image](https://user-images.githubusercontent.com/87197412/235699181-0b566a79-7607-4710-b3c2-a2f49aa1b997.png)

     - ### FIGMA DESIGN
     
 
    ![image](https://user-images.githubusercontent.com/87197412/235700182-174bd214-85ba-4b77-902c-a9cb31e756e3.png)